scripts/setup-linux.sh

```
@@ -0,0 +1,74 @@
+    exit 1
+  fi
+fi
+
+# 1) System prerequisites
+# - build-essential/linux-headers: commonly needed when a dependency must compile (e.g., evdev via pynput)
+# - tesseract/leptonica/pkg-config: required to build/link tesserocr on Linux
+$SUDO apt-get update
+$SUDO apt-get install -y \
+  build-essential \
+  python3-dev \
+  linux-headers-$(uname -r) \
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)